zppy/templates/cryosphere.cfg

```diff
@@ -1,5 +1,5 @@
 [e3sm_diags]
-sets = "aerosol_aeronet","aerosol_budget","annual_cycle_zonal_mean","cosp_histogram","diurnal_cycle","enso_diags","lat_lon","meridional_mean_2d","polar","qbo","streamflow","zonal_mean_2d","zonal_mean_2d_stratosphere","zonal_mean_xy"
+sets= "lat_lon","zonal_mean_xy","zonal_mean_2d","polar","cosp_histogram","meridional_mean_2d","enso_diags","qbo","diurnal_cycle","annual_cycle_zonal_mean","zonal_mean_2d_stratosphere","aerosol_aeronet","aerosol_budget"
```

Comment:
`"aerosol_aeronet","aerosol_budget"` throughout this pull request is why we can't just revert #458 -- doing so would remove those two sets from all these lists.

Reply:
"streamflow" is missing